src/sankeyDiagram.ts

```diff
@@ -2038,98 +2036,116 @@ export class SankeyDiagram implements IVisual {
         return pathParams;
     }
 
-    private getSvgPathForBackwardLink(link: SankeyDiagramLink) {
+    private getSvgPathForBackwardLink(link: SankeyDiagramLink, minHeight: number) {
         let pathParams: string = "";
-        const distanceBetweenLinks: number = 3;
-        const distanceFromNodeToLinks: number = 5;
 
-        const fixedLinkHeight = link.height - distanceBetweenLinks;
-        const linkKneeSize: number = link.height;
+        const fixedLinkHeight = link.height - SankeyDiagram.DistanceBetweenLinks;
```

Comment:
type of **_fixedLinkHeight_** variable is absent

Reply:
fixed in this commit https://github.com/microsoft/powerbi-visuals-sankey/pull/161/commits/0f90ad9016883a0d20c0bf7a137e62b2f99357cf